MCprep_addon/world_tools.py

```diff
@@ -142,6 +143,83 @@ def detect_world_exporter(filepath):
 		# we'll assume this is what the OBJ is using
 		obj_header.set_seperated()
 
+def convert_mtl(filepath):
+	"""Convert the MTL file if we're not using one of Blender's built in
+	colorspaces
+
+	Without this, Blender's OBJ importer will attempt to set non-color data to
+	alpha maps and what not, which causes issues in ACES and whatnot where
+	non-color data is not an option.
+
+	Returns:
+		True if success, False if failed
+	"""
+	blender_standard = (
+		"Standard",
+		"Filmic",
+		"Filmic Log",
+		"Raw",
+		"False Color"
+	)
+	mtl = filepath.rsplit(".", 1)[0] + '.mtl'
+	lines = None
+	copied_file = None
+	with open(mtl, 'r') as mtl_file:
+		lines = mtl_file.readlines()
+
+	if bpy.context.scene.view_settings.view_transform not in blender_standard:
```

Comment:
FYI only in my branched, I inverted this logic and returned early, cutting out the amount of indention below.

There it also returns "None" to indicate no conversion happened, which is a useful state imo.